housekeeping/import/import_from_marc.rb

```diff
@@ -1,27 +1,126 @@
-# first argument is the file containing marc records
-# second is the model name
-# third is the offset to start from
-
-valid_models = ["Source", "Publication", "Holding", "Person", "Institution",
-                "Work"]
-valid_models.each do |model|
-  PaperTrail.request.disable_model(model.constantize)
+# Import Marcxml records into Muscat database
+# Run as bin/rails runner housekeeping/import/import_from_marc.rb
+
+VALID_MODELS = ["Source", "Publication", "Person", "Institution"]
```

Comment:
Why is Work removed here?

Reply:
Because I haven't tested them. And I'm unsure about how are they related to Sources, so I prefer to remove them from my first version. They can be added again in the future.

Reply:
I have extensively used it to import Sources, Publications, Person and Institution records, so those are the ones in the list.

Reply:
But the original script had them, and we certainly use these models in RISM.

Reply:
Or I'll do it myself, of course.

Reply:
Done, thanks for the review and noticing it, @ahankinson!